### PR TITLE
Update some examples

### PR DESCRIPTION
**Pull request by sofianehaddad**
- using new API
 - Pay attention to docstrings regarding parametrization of distribution
 - Fix some examples due to new parametrization

 Probably more examples need to be reviewed

### CODE REVIEW

**Comment by mbaudin47** [comment]
In `python_distribution.ipynb`, there is a warning generated by Matplotlib on the contour:

image

File: python/doc/examples/probabilistic_modeling/python_distribution.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "\n",
     "The way to go is inheriting the PythonDistribution class and overload the methods of the Distribution object.\n",
     "\n",
-    "To create a Copula, the user has to overload isCopula() and return True.\n",
+    "To **create a Copula, the user has to overload isCopula() and return True**.\n",
```
Comment (mbaudin47) [inline python/doc/examples/probabilistic_modeling/python_distribution.ipynb:18]: Please remove the unnecessary bold font: the reader has to read the text anyway.

File: python/doc/examples/probabilistic_modeling/python_distribution.ipynb [hunk 2]
```diff
@@ -151,1498 +157,136 @@
...
+   "source": [
+    "And plot the `cdf`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
+    {
```
[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/doc/examples/probabilistic_modeling/python_distribution.ipynb:188]: this is due to `viewer`. Need to be fixed out of this PR. So I disable warnings for the moments.

File: python/doc/examples/probabilistic_modeling/generate_by_inversion.ipynb [hunk 3]
```diff
@@ -115,7 +124,7 @@
    "source": [
     "def weibullQ(argument):\n",
     "    p, alpha, beta = argument\n",
-    "    quantile = alpha*(-np.log1p(-p))**(1/beta)\n",
+    "    quantile = beta*(-np.log1p(-p))**(1/alpha)\n",
```
[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/doc/examples/probabilistic_modeling/generate_by_inversion.ipynb:127]: done

File: python/doc/examples/probabilistic_modeling/generate_by_inversion.ipynb [hunk 4]
```diff
@@ -386,15 +395,9 @@
...
     "curve.setColors([\"blue\"])\n",
     "graph.add(curve)\n",
-    "# Add the horizontal lines\n",
-    "for i in range(n):\n",
-    "    curve = ot.Curve([0.,weibullSample[i,0]],[uniformSample[i,0],uniformSample[i,0]])\n",
-    "    curve.setColor(\"red\")\n",
-    "    curve.setLineStyle(\"dashed\")\n",
-    "    graph.add(curve)\n",
-    "# Add the vertical lines\n",
+    "# Plot dashed horizontal & verticallines\n",
```
[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/doc/examples/probabilistic_modeling/generate_by_inversion.ipynb:398]: fixed

File: python/doc/examples/probabilistic_modeling/python_distribution.ipynb (hunk 1 above)
Comment (sofianehaddad) [reply python/doc/examples/probabilistic_modeling/python_distribution.ipynb:18]: I think it is important to highlight this part as `Copula` class do not exist anymore.
Our future objective is to enrich this example later with a python `Copula`.
By the way, I remove for the moment "this highlight"

**Comment by sofianehaddad** [comment]
Hope this will be integrated to 1.15

**Pull request merged**